adapters/pubmatic/pubmatic.go

```diff
@@ -158,8 +158,7 @@ func (a *PubmaticAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder
 					}
 
 					pbReq.Imp[i].TagID = strings.TrimSpace(adSlot[0])
-					pbReq.Imp[i].Banner.H = openrtb.Uint64Ptr(uint64(height))
-					pbReq.Imp[i].Banner.W = openrtb.Uint64Ptr(uint64(width))
+					pbReq.Imp[i].Banner = assignBannerHeightAndWidth(pbReq.Imp[i].Banner, uint64(height), uint64(width))
```

Comment:
Don't bother with changes in the old interfaces, such as `Call`. That shouldn't be caught by the json test framework.

Reply:
Rolled back